src/DynamoCoreWpf/VisualizationManager/VisualizationManager.cs

```diff
@@ -237,8 +237,7 @@ public void TagRenderPackageForPath(string path)
                     //This is also a target for memory optimisation
 
                     packages.AddRange(
-                        node.RenderPackages.Where(x => x.Tag == path || x.Tag.Contains(path + ":"))
-                            .Cast<RenderPackage>());
+                        node.RenderPackages.Where(x => x.Description == path || x.Description.Contains(path + ":")));
```

Comment:
As Luke notes above, this looks expensive and honestly, I don't know what this code is doing or why...  Please add comments indicating what this is trying to do.